src/compression/RvlCompression.h

```diff
@@ -0,0 +1,19 @@
+// License: Apache 2.0. See LICENSE file in root directory.
```

Comment:
Can compression sub-library follow the naming convention of librealsense? seems like a small change

Reply:
Is it very important for you? we wrote it following Camel-case notation and it is not so small changes: it is about 10 files, with API

Reply:
no, not a blocker by any means

Reply:
Postponed